lib/src/docs/markdown/cmd.rs

```diff
@@ -62,6 +66,21 @@ mod tests {
 
         flag2 long description
 
+        includes a code block:
+
+        ```
+        $ echo hello world
+        hello world
+
+        more code
+        ```
```

Comment:
Be careful about a pre block containing a fenced block

````
<pre>

foo
```
42
```
bar
</pre>
````

You would have to convert it to

`````
````
foo
```
42
```
bar
````
`````